### DISCUSSION TITLE
Error after upgrading to LR 11.5  - "not connected to Lightroom"

### DISCUSSION THREAD

**Discussion by veneamin** [rsjaffe/MIDI2LR#699] [Problems]
Hi!
After upgrading to LR 11.5  got error  "not connected to Lightroom", MIDI2LR version 5.4.0.0. windows 10
Tried to rollback to previous version of MIDI2LR 5.3 - the same error
Firewall is off

**Comment by rsjaffe** [comment]
Open the plugin manager and make sure the plugin is running. Then restart Lightroom. https://lightroomkillertips.com/understanding-the-plug-in-manager/

**Reply by veneamin** [reply]
"installed and running". Reloading plugin or LR doesn't help

**Reply by veneamin** [reply]
Suddenly after computer restarting the problem has gone.
